python_modules/dagster-graphql/dagster_graphql/implementation/pipeline_run_storage.py

```diff
@@ -234,6 +242,10 @@ def _flush_logs_after_delay(self):
             self.log_sequence = LogSequence()
             self.flush_scheduled = False
 
+    def _cleanup(self):
```

Comment:
this makes the failure events logged during exit process right away, in our case we care that we fire the graphql subscription and get the failure information to the dagit UI before we exit